sklearn/ensemble/tests/test_common.py

```diff
@@ -100,7 +100,7 @@ def test_ensemble_heterogeneous_estimators_behavior(X, y, estimator):
 
     # check that set_params() does not add a new attribute
     estimator_new_params = clone(estimator)
-    svm_estimator = SVC() if is_classifier(estimator) else SVR()
+    svm_estimator = LinearSVC() if is_classifier(estimator) else SVR()
```

Comment:
I don't understand why this is needed. `SVC` uses `probability=False` by default so the deprecation warning should not be raised.

Please revert this unneeded change if you agree with my analysis.